src/app/shared/view-issue/view-issue.component.html

```diff
@@ -0,0 +1,95 @@
+<div>
+    <app-issue-title *ngIf="!isIssueLoading" [issue]="issue" (issueUpdated)="updateIssue($event)"></app-issue-title>
+
+    <mat-divider></mat-divider>
+
+    <div class="row">
+      <div class="column left">
+        <!-- Tester's Post -->
+        <app-issue-description [isEditing]="isIssueDescriptionEditing" (changeEditState)="updateDescriptionEditState($event)" 
+          *ngIf="!isIssueLoading && this.isComponentVisible(issueComponentsEnum.TESTER_POST)" 
+          [issue]="issue" title="Description of Issue" (issueUpdated)="updateIssue($event)">
+        </app-issue-description>
+
+        <!-- Team's Response -->
+        <app-issue-response *ngIf="this.isComponentVisible(issueComponentsEnum.TEAM_RESPONSE) && !isCommentLoading && issue.teamResponse" 
+          [issue]="issue" [isEditing]="isTeamResponseEditing" (updateEditState)="updateTeamResponseEditState($event)"
+          attributeName="teamResponse" (issueUpdated)="updateIssue($event)">
+        </app-issue-response>
+
+        <!-- New Team's Response -->
+        <app-new-team-response *ngIf="this.isComponentVisible(issueComponentsEnum.NEW_TEAM_RESPONSE) &&
+          permissions.canCRUDTeamResponse()  && !issue.teamResponse" [issue]="issue"
+          (issueUpdated)="updateIssue($event)">
+        </app-new-team-response>
+
+        <!-- Tutor's Response -->
+        <app-issue-response *ngIf="this.isComponentVisible(issueComponentsEnum.TUTOR_RESPONSE) && !isCommentLoading && issue.tutorResponse" 
+          [issue]="issue" [isEditing]="isTutorResponseEditing" (updateEditState)="updateTutorResponseEditState($event)"
+          attributeName="tutorResponse" (issueUpdated)="updateIssue($event)">
+        </app-issue-response>
+
+        <!-- New Tutor's Response -->
+        <app-tutor-response *ngIf="this.isComponentVisible(issueComponentsEnum.NEW_TUTOR_RESPONSE)
+          && permissions.canCRUDTutorResponse() && !issue.tutorResponse" [issue]="issue"
```

Comment:
`canCRUDTutorResponse` is not a good name

Reply:
@RonakLakhotia this is from the permission service, changing it would require changing a few things in the codebase which I think should be a separate PR as well.

Reply:
okay sure